### PR TITLE
Created inputs for remaining options

### PR DESCRIPTION
**Pull request by colejoh**
Added new stuff for gender, race, etc.

### CODE REVIEW

File: src/pages/Application/ApplicationDropdown.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { changeApplicationFieldValue } from '../../actions/application';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+const ApplicationTextField = ({changeApplicationFieldValue, field, applicationForm, disabled, isLoading, styles, options}) => (
```
Comment (nickysemenza) [inline src/pages/Application/ApplicationDropdown.js]: `ApplicationTextField` -> `ApplicationDropdown`

File: src/pages/Application/ApplicationDropdown.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <select
+      value={applicationForm[field] || ""}
+      onChange={
+        (e) => changeApplicationFieldValue(field,e.target.value)
+      }
+      disabled={(disabled || false) || isLoading}
+      className="dropdown"
+    >
+        { Object.keys(options).map((t,i) => <option key={i} value={t}>{t}</option>) }
+        {/* <option value="grapefruit">Grapefruit</option>
```
Comment (nickysemenza) [inline src/pages/Application/ApplicationDropdown.js]: shhh don't leak that BM 6 is Hawaiian themed ;)

File: src/assets/_application.scss [hunk 3]
```diff
@@ -81,6 +110,59 @@
...
+/* The switch - the box around the slider */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+  margin-top: 8px;
+}
+
+/* Hide default HTML checkbox */
```
Comment (nickysemenza) [inline src/assets/_application.scss:157]: +1 for comments in css 😄

**Review by nickysemenza — COMMENTED** [review]
👍

File: src/pages/Application/ApplicationDropdown.js (hunk 2 above)
Comment (colejoh) [reply src/pages/Application/ApplicationDropdown.js]: lol oops

**Comment by kedarv** [comment]
👍

**Review by nickysemenza — APPROVED** [review]

**Pull request merged**